webui/react/vite-plugin-branding.ts

```diff
@@ -0,0 +1,18 @@
+import type { Plugin } from 'vite';
+
+export const brandHtml = (): Plugin => {
+  return {
+    name: 'brandHtml',
+    async transformIndexHtml(html: string) {
+      if (process.env.DET_BUILD_EE === 'true') {
+        const cheerio = await import('cheerio');
```

Comment:
out of curiosity: why not use [DOMParser](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser) instead of `cheerio`?